src/components/measureEditor/MeasureEditor.test.tsx

```diff
@@ -203,23 +215,29 @@ describe("MeasureEditor component", () => {
       return Promise.reject({ data: { error: "Something bad happened!" } });
     });
 
+    (synchingEditorCqlContent as jest.Mock)
+      .mockClear()
+      .mockImplementation(() => {
+        return "librarydwwd Test2dvh3wd version '0.0.000'";
+      });
+
     const { getByTestId } = renderEditor(measure);
     const editorContainer = (await getByTestId(
       "measure-editor"
     )) as HTMLInputElement;
     expect(measure.cql).toEqual(editorContainer.value);
     fireEvent.change(getByTestId("measure-editor"), {
       target: {
-        value:
-          "library AdvancedIllnessandFrailtyExclusion_QICore4 version '5.0.000'",
+        value: "librarydwwd Test2dvh3wd version '0.0.000'",
       },
     });
     const saveButton = screen.getByRole("button", { name: "Save" });
     userEvent.click(saveButton);
-    const elmTranslationError = await screen.findByText(
-      "Unable to translate CQL to ELM!"
-    );
-    expect(elmTranslationError).toBeInTheDocument();
+    await waitFor(() => {
+      const successMessage = getByTestId("save-cql-success");
```

Comment:
For future reference, `getByTestId` is the lowest recommended query with RTL. See [RTL Query Priority](https://testing-library.com/docs/queries/about#priority).

> getByTestId: The user cannot see (or hear) these, so this is only recommended for cases where you can't match by role or text or it doesn't make sense (e.g. the text is dynamic).

Reply:
Thanks Joe, I'll keep that in mind